pep-0458.txt

```diff
@@ -819,7 +819,13 @@ include a version number in their filename:
   VERSION_NUMBER.ROLENAME.json,
     where VERSION_NUMBER is an incrementing integer, and ROLENAME is one of the
     top-level metadata roles -- *root*, *snapshot* or *targets* -- or one of
-    the delegated targets roles -- *bins* or *bin-n*.
+    the delegated targets roles -- *bins* or *bin-n*. The version number for
+    *snapshot*, *targets*, *bins*, and *bin-n* roles may eventually loop back to 0 as long
+    as the previous 0-version metadata has expired before this occurs. To
+    prevent confusion, we suggest retiring the oldest metadata for each non-root
+    metadata type when 0.5*MAX_INT versions are in use. Root metadata can
+    not be retired, so the root metadata expiration should be set such that a
+    version number overflow is unlikely (ex. once a year).
```

Comment:
I'd phrase this the other way around, that is root metadata bumps so rarely (once a year due to expiration, and after hopefully rare incidents like a compromise) that integer overflow is not an issue for root metadata. But you can also leave it as it is.